The tests expect a tablespace to exists, we let the tests create it

--- a/regress/sql/init-extension.sql
+++ b/regress/sql/init-extension.sql
@@ -1,3 +1,7 @@
 SET client_min_messages = warning;
 CREATE EXTENSION pg_repack;
 RESET client_min_messages;
+CREATE TABLESPACE testts LOCATION '/tmp/pg-repack-testts';
+CREATE TABLESPACE "test""ts" LOCATION '/tmp/pg-repack-test"ts';
+CREATE TABLESPACE "test ts" LOCATION '/tmp/pg-repack-test ts';
+CREATE TABLESPACE "1testts" LOCATION '/tmp/pg-repack-1testts';
--- a/regress/expected/init-extension.out
+++ b/regress/expected/init-extension.out
@@ -1,3 +1,7 @@
 SET client_min_messages = warning;
 CREATE EXTENSION pg_repack;
 RESET client_min_messages;
+CREATE TABLESPACE testts LOCATION '/tmp/pg-repack-testts';
+CREATE TABLESPACE "test""ts" LOCATION '/tmp/pg-repack-test"ts';
+CREATE TABLESPACE "test ts" LOCATION '/tmp/pg-repack-test ts';
+CREATE TABLESPACE "1testts" LOCATION '/tmp/pg-repack-1testts';
